src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1634,6 +1648,14 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
             ResolutionContext resolutionContext, INuGetProjectContext nuGetProjectContext,
             IEnumerable<SourceRepository> primarySources, IEnumerable<SourceRepository> secondarySources,
             CancellationToken token)
+        {
+            return await PreviewInstallPackageAsync(nuGetProject, packageIdentity, resolutionContext, nuGetProjectContext, primarySources, secondarySources, versionRange: null, token);
+        }
+
+        public async Task<IEnumerable<NuGetProjectAction>> PreviewInstallPackageAsync(NuGetProject nuGetProject, PackageIdentity packageIdentity,
+            ResolutionContext resolutionContext, INuGetProjectContext nuGetProjectContext,
+            IEnumerable<SourceRepository> primarySources, IEnumerable<SourceRepository> secondarySources,
+             VersionRange versionRange, CancellationToken token)
```

Comment:
I'm so used to method arguments being on a single line, or one per line, I find this a little difficult to read. I don't know how the rest of the team feels, but I'd personally prefer it to be one per line. Plus make sure the last line uses the same indenting as the other lines.